service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantController.java

```diff
@@ -1,43 +1,68 @@
 package uk.gov.hmcts.reform.fpl.controllers;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import feign.FeignException.NotFound;
 import io.swagger.annotations.Api;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
 import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
 import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
 import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.request.RequestData;
 import uk.gov.hmcts.reform.fpl.service.ApplicantService;
 import uk.gov.hmcts.reform.fpl.service.UpdateAndValidatePbaService;
+import uk.gov.hmcts.reform.rd.client.OrganisationApi;
+import uk.gov.hmcts.reform.rd.model.Organisation;
 
 @Api
+@Slf4j
 @RestController
 @RequestMapping("/callback/enter-applicant")
 public class ApplicantController {
 
     private final ApplicantService applicantService;
     private final UpdateAndValidatePbaService updateAndValidatePbaService;
     private final ObjectMapper mapper;
+    private final AuthTokenGenerator authTokenGenerator;
+    private final OrganisationApi organisationApi;
+    private final RequestData requestData;
 
     @Autowired
     public ApplicantController(ApplicantService applicantService,
                                UpdateAndValidatePbaService updateAndValidatePbaService,
-                               ObjectMapper mapper) {
+                               ObjectMapper mapper,
+                               OrganisationApi organisationApi,
+                               AuthTokenGenerator authTokenGenerator,
+                               RequestData requestData) {
         this.applicantService = applicantService;
         this.updateAndValidatePbaService = updateAndValidatePbaService;
         this.mapper = mapper;
+        this.organisationApi = organisationApi;
+        this.authTokenGenerator = authTokenGenerator;
+        this.requestData = requestData;
     }
 
     @PostMapping("/about-to-start")
     public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody CallbackRequest callbackrequest) {
         CaseDetails caseDetails = callbackrequest.getCaseDetails();
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
-        caseDetails.getData().put("applicants", applicantService.expandApplicantCollection(caseData));
+        Organisation organisation = Organisation.builder().build();
```

Comment:
could you add the builder into that catch block? Makes it clear where and why this is being done

Reply:
i've done this now :) makes sense